src/Common/src/Interop/Windows/Winsock/Interop.WSAGetOverlappedResult.cs

```diff
@@ -11,11 +11,11 @@ internal static partial class Interop
     internal static partial class Winsock
     {
         [DllImport(Interop.Libraries.Ws2_32, SetLastError = true)]
-        internal static extern bool WSAGetOverlappedResult(
-            [In] SafeCloseSocket socketHandle,
-            [In] SafeHandle overlapped,
-            [Out] out uint bytesTransferred,
-            [In] bool wait,
-            [Out] out SocketFlags socketFlags);
+        internal static unsafe extern bool WSAGetOverlappedResult(
+            SafeCloseSocket socketHandle,
+            SafeHandle overlapped,
+            uint* bytesTransferred,
```

Comment:
Beyond the issues we already talked about, there's one other logical disadvantage to using a pointer instead of out in a situation like this: the caller doesn't know whether the location needs to be initialized prior to the call.